packages/react/src/components/masthead.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { PropsWithChildren, useEffect } from "react";
+import { ChevronDownIcon } from "../icons/chevron-down";
+import { Lock2Icon } from "../icons/lock-2";
+import SolidLockIcon from "../icons/solid-lock";
+import MalaysiaFlagIcon from "../icons/malaysia-flag";
+import { GovMyIcon } from "../icons/gov-my";
+
+// Types
+type MastheadContentProps = {
+  icon: React.ReactNode;
+  children: React.ReactNode;
+};
+
+type MastheadSectionTitleProps = {
+  children: React.ReactNode;
+};
+
+type MastheadSectionBodyProps = {
+  children: React.ReactNode;
+};
+
+// Component definitions
+export const MastheadHeader = ({ children }: PropsWithChildren) => {
```

Comment:
1. use `FunctionComponent<...>` generic when defining components.
2. Follow this pattern;
```tsx
// basic component ✅
const ComponentA: FunctionComponent<...> = ({ ... }) => <CompA />

// forwardRef component ✅
const ComponentB: ForwardRefExoticComponent<...> = (({ ... }, ref) => <CompB />)

// exports ✅
export { ComponentA, ComponentB }

// export + component  ❌
export const ComponentC = () => ...
```

Reply:
followed the pattern as described